broker/openshift-origin-broker.spec

```diff
@@ -79,6 +79,7 @@ Requires:      %{?scl:%scl_prefix}rubygem-metaclass
 Requires:      %{?scl:%scl_prefix}rubygem-mime-types
 Requires:      %{?scl:%scl_prefix}rubygem-moped
 Requires:      %{?scl:%scl_prefix}rubygem-multi_json
+Requires:      %{?scl:%scl_prefix}rubygem-net-ssh
```

Comment:
This dependency should actually exist on the controller not the broker, so move this to controller/openshift-origin-controller.gemspec